### PR TITLE
fixing small bugs

### PR DESCRIPTION
**Pull request by lucaslin**
updating maxScore
fixing typo

### CODE REVIEW

File: Test/MainWindow.xaml.cs [hunk 1]
```diff
@@ -75,7 +75,7 @@ private void ButtonClick(object sender, RoutedEventArgs e)
 		protected virtual void OnPropertyChanged([CallerMemberName] string propertyName = null)
 		{
 			PropertyChangedEventHandler handler = PropertyChanged;
-		    handler?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+		    handler.Invoke(this, new PropertyChangedEventArgs(propertyName));
```
Comment (bertyhell) [inline Test/MainWindow.xaml.cs:78]: This is a conditional operator since handler could be null. Why did you remove this? 
http://www.kunal-chowdhury.com/2014/12/csharp-6-null-conditional-operators.html?m=1

Comment (lucaslin) [inline Test/MainWindow.xaml.cs:78]: Sorry. Thought it was a typo  (new to C#).

**Comment by bertyhell** [comment]
Thank you for the PR btw. It's the first one I've ever gotten for one of my free time projects. :D
